om.lic

```diff
@@ -0,0 +1,82 @@
+
+custom_require.call(%w[common common-arcana common-travel])
+
+class CreateOsrelMeraud
+  include DRC
+  include DRCT
+
+  def initialize
+    unless DRStats.cleric?
+      echo "You're not a cleric!"
+      exit
+    end
+      
+    if DRSpells.active_spells.include?('Osrel Meraud')
+      echo "OM already active!"
+      exit
+    end
+
+    @settings = get_settings
+
+    unless @settings.waggle_sets['create_om']
+      echo "This script uses waggle_sets, you need a create_om. It will also buff with a buff_om set."
+      exit
+    end
+
+    echo "No OM found!"
+    
+    buff_om if create_om if get_orb
+  end
+
+  def get_orb
+    case bput("tap my #{@settings.osrel_favor_god} orb", 'I could not find', 'The orb is delicate')
+    when 'The orb is delicate'
+      echo "Already have an orb for #{@settings.osrel_favor_god}!"
```

Comment:
could we just reuse @settings.imortal_aspect for this instead of making a new setting? that settings only purpose was to carve beads for om orbs in theurgy.lic.

Reply:
Well, the issue we have with that is that immortal_aspect is asking for the creature, not the god. In this instance, I am wanting to keep all things together with the same set of names, `osrel_`. 

```
osrel_favor_god: Firulf
osrel_no_harness: true
osrel_amount: 15
```
I think in this game, what should be done is that theurgy should use the `osrel_favor_god` and then use the base-theurgy to find the creature/aspect based on the god's name.

What do you think?

Reply:
I mean, that's fine, it's just more settings when we already have a setting for it and I thought maybe we could piggy back on whats already there instead of adding more settings. This script could just do the conversion alternately the same way favor.lic does, the conversion is available in base-theurgy already. I'm not invested in either option just wanted to bring it up :)

Reply:
I actually definitely agree with this, but I think it should be the reverse of what you are suggesting, or it shouldn't be implemented at all. Going from creature to immortal name just seems backwards to me within this type of situation, and in the future would be one of those "why did I code it this way" type of things. And then anyone using it would be like "it's trying to find a firfulf orb, why am I putting in owl? Again, I completely agree and I thought of this, but it's just too far backwards for me.